app/src/main/java/com/twilio/video/app/ui/room/RoomViewEvent.kt

```diff
@@ -3,7 +3,7 @@ package com.twilio.video.app.ui.room
 import com.twilio.audioswitch.selection.AudioDevice
 
 sealed class RoomViewEvent {
-    object ScreenLoad : RoomViewEvent()
+    object RefreshViewState : RoomViewEvent()
```

Comment:
Should there be an explicit state for the permissions result?

Reply:
Yeah there definitely should be. I haven't quite gotten to refactoring the permissions code along with local participant management.